src/api/apiHelpers.test.ts

```diff
@@ -392,6 +391,21 @@ describe('[api] helpers', () => {
       await expect(getResult).rejects.toThrow(error)
     })
 
+    it.each([
+      500, // Internal Server Error
+      502, // Bad Gateway
+      503, // Service Unavailable
+      504, // Gateway Timeout
+    ])(
+      'should not throw error when status is not ok and is an error code whose exception is not captured',
```

Comment:
j'avoue que je comprends pas trop la description du test 😬  je me serais attendu à qqch du genre : 
should not throw error if status is 5xx

Reply:
J'ai update le nom